apps/docs-app/docs/integrations/ionic/index.md

```diff
@@ -210,6 +208,91 @@ export default defineConfig(({ mode }) => {
 @import '@ionic/angular/css/palettes/dark.system.css';
 ```
 
+### Server Side Rendering Caveat
+
+Ionic Framework [doesn't support the Angular's new Client Hydration](https://github.com/ionic-team/ionic-framework/issues/28625#issuecomment-1843919548), as `@angular/ssr` [doesn't support web components](https://github.com/angular/angular/issues/52275), and when they are supported, work has to be done on the Stencil components to enable it. So right now there are three ways to handle this:
```

Comment:
We don't use the Angular SSR package, so this could be a little more clear as an Angular limitation.

```suggestion
Ionic Framework [doesn't support the Angular's new Client Hydration](https://github.com/ionic-team/ionic-framework/issues/28625#issuecomment-1843919548), as `Angular` [doesn't support web SSR with components](https://github.com/angular/angular/issues/52275), and when they are supported, work has to be done on the Stencil components to enable it. So right now there are three options to handle this:
```

Reply:
Updated with your comments @brandonroberts